### PR TITLE
community[patch]: add clientParams option to Chroma vectorstore init args

### LINKED ISSUE #6296: Allow Auth w/ Chroma
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

I do not see any way to add auth via the chroma wrapper here:
https://github.com/langchain-ai/langchainjs/blob/993ce7a822bd11e125419bff84a9951bd58e66e5/langchain/src/vectorstores/chroma.ts#L109C1-L112C8

### Error Message and Stack Trace (if applicable)

_No response_

### Description

The following issue was closed, but doesn't seem like it was ever fixed
https://github.com/langchain-ai/langchainjs/issues/3441

### System Info

mac, osx, latest version of node, etc...

### PR DESCRIPTION
**Pull request by jeasonnow**
Fixes #6296

### CODE REVIEW

File: libs/langchain-community/src/vectorstores/chroma.ts [hunk 1]
```diff
@@ -67,12 +73,16 @@ export class Chroma extends VectorStore {
     return "chroma";
   }
 
-  constructor(embeddings: EmbeddingsInterface, args: ChromaLibArgs) {
+  constructor(
+    embeddings: EmbeddingsInterface,
+    args: ChromaLibArgs & { clientParams?: Omit<ChromaClientParams, "path"> }
```
Comment (bracesproul) [inline libs/langchain-community/src/vectorstores/chroma.ts:78]: Could you instead make this a single exported interface `ChromaArgs`

Comment (bracesproul) [reply libs/langchain-community/src/vectorstores/chroma.ts:78]: Actually, just update `ChromaLibArgs` to include this `clientParams` field in both options of the type. That should be enough

**Review by bracesproul — CHANGES_REQUESTED** [review]
One small comment, otherwise this looks good

**Comment by jacoblee93** [comment]
Thanks for this!

@bracesproul before we ship a new community version with this can we make sure there's no peer dep conflicts from `npm install`?

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**